### PR TITLE
Fix messaging

### PR DESCRIPTION
**Pull request by tsujigiri**

### CODE REVIEW

File: spec/features/messaging_spec.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+RSpec.feature 'Messaging' do
```
Comment (philippbayer) [inline spec/features/messaging_spec.rb]: it still does it :D

Comment (tsujigiri) [inline spec/features/messaging_spec.rb]: I think it's because I don't have the current config in this branch.

Comment (philippbayer) [inline spec/features/messaging_spec.rb]: Aaah yes, that makes sense

**Comment by tsujigiri** [comment]
I wasn't done...

**Comment by philippbayer** [comment]
Oops sorry Travis passed all tests, shouldn't new commits be added here?

**Comment by tsujigiri** [comment]
Don't worry. Unless you don't trust the lack of test coverage... :see_no_evil:

**Comment by tsujigiri** [comment]
It's only fixing the Hound complaints.

**Pull request merged**